### ISSUE TITLE
New Try // Broken Layout

### ISSUE THREAD

**Issue by beastyrabbit** [RafaelDeJongh/MyAnimeList-Modern-RDJ-Theme#1]
First of all sorry for posting in the wrong Repo. [Issue](https://github.com/RafaelDeJongh/MyAnimeList-Classic-RDJ-Theme/issues/4)
But the supposed solution is not fixing my Problem. 

After looking into the Problem it seems that Issue is that in the Class= "data image" the height is set to auto. When I change it to a fixed Value everything works wonderfully.
On auto: https://i.imgur.com/kKf8EtA.png
On 335px: https://i.imgur.com/wW9etsS.jpg

Best regards

**Comment by RafaelDeJongh** [comment]
Normally that should have a min-height, could you link me your profile please?

**Comment by beastyrabbit** [comment]
Hey,
my user profile [Link](https://myanimelist.net/animelist/Beastyrabbit?status=6)
You have to have [this](https://greasyfork.org/de/scripts/372847-mal-sync) Extention installed.

**Comment by RafaelDeJongh** [comment]
Oh so it's only with that extension installed, well I do read on the page you linked that it mentions this: 

> You will also have larger thumbnails on MAL, with options to choose your preferred size from a settings-tab in miniMAL.

So see if this could be fixed via the settings tab, I bet your problem has something to do with that.

**Comment by beastyrabbit** [comment]
Oh sorry, I thought you know the extension. (I thought that it is widely spread and used)
I should have explained in more detail. 
The extension removes the dimensions of the Pictures 
[Before](https://myanimelist.cdn-dena.com/r/96x136/images/anime/2/88384.webp?s=c8f85f5c1604c05d18c742ec0863600c)
[After](https://myanimelist.cdn-dena.com/images/anime/2/88384.webp?s=c8f85f5c1604c05d18c742ec0863600c)
So you get nicely HD Pictures. But the Original Pictures are not all the same size. 
I was hoping you can give me an option to keep the HD Pictures and your awesome design. 
The Settings-tab is not doing what I want 

````javascript 
  bookmarksHDimages() {
        var tags = document.querySelectorAll('img[src*="/96x136/"]');
        for (var i = 0; i < tags.length; i++) {
            var regexDimensions = /\/r\/\d*x\d*/g;
            var url = tags[i].getAttribute("src");
            tags[i].setAttribute("src", url.replace(regexDimensions, ''));
        }
    }
````

That peace of code is makeing the pictures HD

**Comment by RafaelDeJongh** [comment]
Never heard or used it so no clue what's up with that.
The instructions to my theme does also include custom HD covers compared to MAL's original.
The function you got there pretty much just looks like it is going to to change the url to not be limited to the currently set low quality versions of the images. 

Easiest way as this is more of a personal/third party issue it would be to remove that function from the script.

But I installed the script myself to see if I could change anything and I've uploaded a new version of the theme that normally shouldn't interfere for users who don't use MAL-Sync and function as it should for people who do. 

Let me know if it works for you, I will probably soon switch to CSS Grids instead of still using % to get my list order, but that wouldn't change much to this problem either.

Either way thanks for the report and feel free to let me know if it works for you or not!
